mama/c_cpp/src/gunittest/c/MainUnitTestC.cpp

```diff
@@ -60,16 +84,36 @@ static void parseCommandLine (int argc, char** argv)
 
     for (i = 1; i < argc;)
     {
-		if (strcmp ("-m", argv[i]) == 0)
-		{
-			gMiddleware = argv[i+1];
-			i += 2;
-		}
-		else if (strcmp ("-p", argv[i]) == 0)
-		{
-			gPayload = argv[i+1];
-			i += 2;
-		}
+        if (strcmp ("-m", argv[i]) == 0)
+        {
+            gMiddleware = argv[i+1];
+            i += 2;
+        }
+        else if (strcmp ("-p", argv[i]) == 0)
+        {
+            gPayload = argv[i+1];
+            i += 2;
+        }
+        else if (strcmp ("-tport", argv[i]) == 0)
```

Comment:
I'm more referring to the new ones - the -m and -p ones were there already and are used to specify (for example) -m qpid -p qpidmsg so the tests can run on open source middlewares. As long as they're not mandatory for other middlewares to run though it's OK for now though.